extension/src/popup/components/AutoSave/styles.scss

```diff
@@ -6,7 +6,7 @@
     position: fixed;
     top: 0;
     width: 100%;
-    transition: opacity var(--anim-transition-default);
+    transition: opacity var(--dropdown-animation);
```

Comment:
apparently this var doesn't exist in the repo anymore, so replacing it with this other animation timing var